.travis.yml

```diff
@@ -9,9 +9,12 @@ stages:
 python: '3.7'
 cache: false
 
+# token to upload github release notes
 env:
   global:
     - PYTEST_ADDOPTS=-vv
+    # used by publish_gh_release_notes
+    - secure: "0ldsOQ5B7bFuOQl6ovvttl3dW9zDh+YG7NjKTESlKGfdzNgWnuNIEoAfu7qj8r0RCXmF4nejQM5X+KhTIdWv4NCNFGSB3QHEQ82vltuh4A400vYj76DLg7YWupFAGyNWwtsi/DPAsIoLkjoTcX8KgwYqqMx6wrNGQ9qz8VWQn+BDtg/A98D33ACpZyNLxQONQjY+EIqmS50XMkJLci+929tjA9xMC2DQCEIzOkDhJhcclGHJKY8tvhjPUGUEmNPk9KWOacaaPSroLvLAEnn28s92V3Haf8+2kkff5/cpgWLzUtf9klx72LXNQ5+1mTTxAJcN3Zxxk5ci1UCQh4hjFc2/xndf5wjPrcwlyf/q5BM/PXg5a4bJfBOV2eVlLC97KfP9MZThlxplMbZSHQLMbCFrWlWztL6xDbOHialfUq0nQmiq5gd7e0izwBXDmSbWGOMvpKutdntBJGAtn6tFO+wDR4jrPwT2sCZYOt10wQ8jiePw/30hN5bjv84+YpSyEQ6ZIK9bnx9Mo+142wXQbsjaf1YRBLsT4flGByKomR2HTf1vs++8JZmdHU60JY9YR/HYFd6wPji5Dk1Qd6bFILFXtqn3hGi3KRZzJ9fVVsQ2vfwbKBcux8pDnzTNioGDvqDRBSJreQhDnVlxcavK6vpVsAPb/VdAoVaUSlhZZAg="
```

Comment:
I've tried to put this token only in the `deploy` stage like this:

```yaml
    - stage: deploy
      python: '3.6'
      ...      
      env:
        global:
          - secure: "ZpcZbxgxp2AmB9o9rC57/wkQDgfkDzqLsPrf72oNU
```

But this failed for some reason:

```
$ export PYTEST_ADDOPTS=-vv
$ export global={:secure=>
/home/travis/.travis/functions: eval: line 104: syntax error near unexpected token `newline'
/home/travis/.travis/functions: eval: line 104: `export global={:secure=> '
```

(Build: https://travis-ci.org/nicoddemus/pytest/builds/570222582)

Reply:
Would not be "global" there.. ;)

Reply:
i.e., try:
```diff
diff --git i/.travis.yml w/.travis.yml
index af33d672e..668614d71 100644
--- i/.travis.yml
+++ w/.travis.yml
@@ -74,6 +74,8 @@ jobs:
       python: '3.6'
       install: pip install -U setuptools setuptools_scm
       script: skip
+      env:
+        - secure: …
       deploy:
         provider: pypi
         user: nicoddemus
```